pkg/gatewayserver/redis/registry.go

```diff
@@ -86,3 +87,45 @@ func (r *GatewayConnectionStatsRegistry) Get(ctx context.Context, ids *ttnpb.Gat
 	}
 	return result, nil
 }
+
+// BatchGet returns the connection stats for a batch of gateways.
+// NotFound errors indicating that the gateway is either not connected
+// or is connected to a different cluster, are ignored.
+func (r *GatewayConnectionStatsRegistry) BatchGet(ctx context.Context,
+	ids []*ttnpb.GatewayIdentifiers,
+) ([]*ttnpb.GatewayConnectionStatsEntry, error) {
+	ret := make([]*ttnpb.GatewayConnectionStatsEntry, 0)
```

Comment:
Very minor nit: Since you know the length of the result, both of those can become `make([]*ttnpb.GatewayConnectionStatsEntry, 0, len(ids))` and `make([]string, 0, len(ids))`